rest/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id("org.springframework.boot") version "3.1.3"
+    id("io.spring.dependency-management") version "1.1.3"
+    kotlin("jvm") version "1.9.0"
+    kotlin("plugin.noarg") version "1.9.0"
+    kotlin("plugin.spring") version "1.9.0"
+    id("io.gitlab.arturbosch.detekt") version ("1.23.1")
+}
```

Comment:
Consider specifying version in top level build file, so in all the submodules you won't need to so. (If you have more modules than now).